jinja2_git.py

```diff
@@ -11,14 +11,27 @@ class GitExtension(Extension):
 
     tags = {'gitcommit'}
 
-    def _commit_hash(self):
-        output = subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+    def _commit_hash(self, *args):
+        if args[0]:
+            output = subprocess.check_output(['git',
+                                              'rev-parse',
+                                              '--short',
+                                              'HEAD',
+                                              ])
+        else:
+            output = subprocess.check_output(['git', 'rev-parse', 'HEAD'])
```

Comment:
Please, try to avoid duplicating the command.

Reply:
What is the best way to avoid duplicating here? Bring the list to a variable?

Reply:
I can thing about `insert()` call when `short is True`.
There are other options as well.